documentation/airport-format.md

```diff
@@ -1,309 +1,228 @@
+        "base": [
+            ["N36d38m01.199", "W114d36m17.219", "N36d36m32.337", "W114d34m19.673"],
+            ["N36d36m27.904", "W114d36m12.534", "N36d38m06.271", "W114d34m20.227"],
+            ["N35d56m01.371", "W114d51m25.735", "N35d57m09.977", "W114d51m43.334"],
+            ["N35d56m42.691", "W114d52m17.075", "N35d56m28.981", "W114d50m51.994"]
+        ]
+    }
 }
 ```
 
-For `lat, lon, elevation` fields, you can either use the standard `[x, y]`
-notation or the new `["LAT", "LON", "ELEVATION"]` notation. The latter uses this
-format:
-
-    ["N or S followed by a latitude", "W or E followed by a longitude", "optional altitude ending in 'ft' or 'm'"]
-where latitude and longitude are numbers that follow this format:
-    <degrees>[d|°][<minutes>m[<seconds>s]]
-
-Examples of acceptable positions:
-  [  40.94684722   ,  -76.61727778   ], // decimal degrees
-  [ "N40.94684722" , "W76.61727778"  ], // decimal degrees
-  [  "N40d56.811"  ,  "W076d37.037   ], // degrees, decimal minutes
-  [ "N40d56m48.65" , "W076d37m02.20" ]  // degrees, minutes, decimal seconds
-
-If you use a `["lat", "lon"]` combination for any `position`, you
-_must_ set the `position` of the airport as well; if you use `end`,
-`length` and `angle` are not needed (but can be used to override
-`end`).
-
-Aircraft scheduling
--------------------
-The 'type' key in an arrival or departure block determines the algorithm
-that will be used to spawn the aircraft. Each type take slightly different
-parameters in order for you to shape the airport's traffic to your liking.
-
-
-## Bare Minimum Elements
+## Comments
 
-At the very least, an arrival stream MUST have definitions for the
-following parameters. Additional may be required if the spawn method
-is set to one other than 'random'.
+### Property Descriptions
 
-            BARE MINIMUM PARAMETERS:
-   PARAMETER   REQ      PARAMETER DESCRIPTION
-+-------------+---+-------------------------------+
-| 'airlines'  | * | weighted array of airlines    |
-+-------------+---+-------------------------------+
-| 'altitude'  | * | altitude to spawn at          |
-+-------------+---+-------------------------------+
-| 'frequency' | * | spawn rate, aircraft per hour |
-+-------------+---+-------------------------------+
-| 'heading'   | * | heading to fly on spawn       |
-|    (OR)     |   |                               |
-|  'fixes'    | * | array of fixes to go to       |
-|    (OR)     |   |                               |
-|  'route'    | * | properly formatted route*     |
-+-------------+---+-------------------------------+
-| 'speed'     | * | speed to spawn at (knots)     |
-+-------------+---+-------------------------------+
-  *see index.md for route format (ex: 'BSR.BSR1.KSFO')
+#### Latitude, Longitude, Elevation
 
+For `lat, lon, elev` values, these formats are acceptable:
+    [40.94684722, -76.61727778, "866ft"],
```

Comment:
These should be an unordered list with bat ticks, then you can also remove the commas:

- `[40.94684722, -76.61727778, "866ft"]`